File: app/src/main/java/com/upence/data/Categories.kt
package com.upence.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity()
data class Categories(
    @PrimaryKey(autoGenerate = true)
    val id: Int,
    val icon: String,
    val name: String,
    val color: Long,
    val description: String,
)

File: lib/data/database/tables/categories.dart
import 'package:drift/drift.dart';

@DataClassName('Category')
class Categories extends Table {
  IntColumn get id => integer().autoIncrement()();
  TextColumn get name => text()();
  TextColumn get icon => text()();
  IntColumn get color => integer()();
  TextColumn get description => text().nullable()();
}
